src/code42cli/cmds/securitydata.py

```diff
@@ -97,7 +98,25 @@
 file_category_option = click.option(
     "--file-category",
     multiple=True,
-    type=click.Choice(list(FileCategory.choices())),
+    type=MapChoice(
+        choices=list(FileCategory.choices()),
+        extras_map={
```

Comment:
I got really annoyed with this similarly in the XSOAR integration and just made it always work with any cases or underscore/hyphen combo using this logic: https://github.com/code42/content/blob/master/Packs/Code42/Integrations/Code42/Code42.py#L530-L544

```
def get_file_category_value(key):
    # Meant to handle all possible cases
    key = key.lower().replace("-", "").replace("_", "")
    category_map = {
        "sourcecode": FileCategory.SOURCE_CODE,
        "audio": FileCategory.AUDIO,
        "executable": FileCategory.EXECUTABLE,
        "document": FileCategory.DOCUMENT,
        "image": FileCategory.IMAGE,
        "pdf": FileCategory.PDF,
        "presentation": FileCategory.PRESENTATION,
        "script": FileCategory.SCRIPT,
        "spreadsheet": FileCategory.SPREADSHEET,
        "video": FileCategory.VIDEO,
        "virtualdiskimage": FileCategory.VIRTUAL_DISK_IMAGE,
        "archive": FileCategory.ZIP,
    }
    return category_map.get(key, "UNCATEGORIZED")
```
Just thought I'd share because this way we are resilient to any mapping changes.  There have been breaking changes in that past that broke the XSOAR integration on this specific mapping

Reply:
this approach also gives some additional flexibility like allowing `pdf` as valid in addition to `PDF` and `Pdf`.

Reply:
this would also mean `__p__d__f___` will be a valid value too, is that ok?

Reply:
I don't see why not, unless we think that there's a chance that could become a name for a different category.

Reply:
i.e. would we ever have two categories where the only difference in the names is the presence of underscores?

Reply:
Lol. Would multiple underscores/hyphens really work? Values like `P-D_F` would work and that is odd but would have to be _discovered_ accidentally or something and even then, wouldn't have too surprising of an effect.

But would multiple underscores/hyphens really work with this advanced mapper suggestion? That would mean values like `P-_---_--Df` would work?

Reply:
actually, even then it wouldn't matter because that would be a new category that we wouldn't do this mapping on

Reply:
Oh wait I see now how it works.

Reply:
If we normalize and allow any casing/mix of underscores here, we should probably do it everywhere, no? It'd be odd to allow `sourcecode` on this one option when the choice lists `SourceCode` but not allow `--type applicationread`.

Reply:
@timabrmsn standardizing that sounds like a separate effort -- let's keep this as-is for now since it addresses the main problem.